src/main/java/com/box/sdk/BoxUser.java

```diff
@@ -308,6 +309,34 @@ public void updateInfo(BoxUser.Info info) {
         JsonObject jsonObject = JsonObject.readFrom(response.getJSON());
         info.update(jsonObject);
     }
+    
+        /**
+     * Moves all of the owned content from within one user’s folder into a new folder in another user's account.
+     * You can move folders across users as long as the you have administrative permissions and the 'source' 
+     * user owns the folders. Per the documentation at the link below, this will move everything from the root 
+     * folder, as this is currently the only mode of operation supported.
+     * 
+     * See also https://box-content.readme.io/reference#move-folder-into-another-users-folder
+     * 
+     * @param sourceUserId the user id of the user whose files will be the source for this operation
```

Comment:
Capitalize ID. This is just a stylistic thing we try to be consistent with this throughout the SDK.